tests/scripts/other_langs/r_package_streamlined_fail/config.yaml

```diff
@@ -0,0 +1,40 @@
+optimization_options:
```

Comment:
It would be good to add comments in this config file to mark which parts will cause the run to fail

Reply:
Sounds good.